database/methods/delete.py

```diff
@@ -0,0 +1,23 @@
+from .base_query import BaseQuery
+from database import Constants, DBFunction
+
+
+class Delete(BaseQuery):
+    def __init__(self, table_name, where=None):
```

Comment:
isn't this dangerous, what happens if invoked and where condition is None?

basically want to avoid delete from table x; 

delete operations becomes a bit more dangerous, so have to be more careful.

Reply:
then it would delete everything from the table x. seeing as no one except the devs should have access to this, i dont see a problem with it